### DISCUSSION TITLE
How to set input node path url instead of uploading file

### DISCUSSION THREAD

**Discussion by Guiwoo** [comfyanonymous/ComfyUI#2834] [Q&A]
Hi guys i would like to send input image via aws-s3 but when i send to prompt json body (just fix input file name to URL path)
I got custom validate error 

ERROR:root:  - Custom validation failed for node: image - Invalid image file: URL path
ERROR:root:Output will be ignored
invalid prompt: {'type': 'prompt_outputs_failed_validation', 'message': 'Prompt outputs failed validation', 'details': '', 'extra_info': {}}

It seems like look up the input folder, But do not wanna upload on input folder path 
Bring aws-s3-image as stream data and generate image 
Please let me know where do I change script ?

**Comment by ltdrdata** [comment]
FYI, https://github.com/jitcoder/lora-info provides `ImageFromURL` node.

**Reply by Guiwoo** [reply]
Thank you so much
